modules/install/pages/ipv6-setup.adoc

```diff
@@ -22,14 +22,35 @@ See xref:install-intro.adoc[Installing the Couchbase Server Cluster] for more in
 == IPv6 Set-Up for Linux
 
 After the Couchbase Server package has been installed, the `couchbase` service is started by default.
-To enable IPv6, proceed as follows:
+Select the appropriate set-up procedure from the two procedures provided below, according to the cluster’s Couchbase _server-version_.
+
+=== Version 6.0.0
+
+Proceed as follows:
 
 . Stop the `couchbase-server` service.
 . In `/opt/couchbase/etc/couchbase/static_config`, set `ipv6` to `true`.
 . Delete `/opt/couchbase/var/lib/couchbase/config/config.dat`.
 . Restart the `couchbase-server` service (this starts Couchbase Server in IPv6 mode).
 To check, point a supported web browser to  `http://[::1]:8091`, which is the IPv6 address and port number on which Couchbase Server should be running.
 
+This concludes the procedure.
+
+=== Version 6.0.1
```

Comment:
Perhaps we should and "and higher".
=== Version 6.0.1 and higher

Reply:
On the other hand we don't want this to be used for 6.5.
We can use something like "and all next 6.0.* version"

Reply:
ok